code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -618,3 +618,412 @@ im not even gonna bother with these for the following symptoms. typed em out, co
+				slowdown = (min(3, slowdown + efficacy))
+				if((M.getToxLoss() && toxheal) || isslimetarget(M))
+					M.adjustToxLoss(-efficacy)
+					if(grub.growthstage < (A.stage - 2))
+						grub.food += 1 * power
+					grub.patience = (rand(60, 120) / power)
+				else
+					grub.patience --
+			if(((M.getToxLoss() > (LAZYLEN(grubs) * (30/power))) || isslimetarget(M)) && prob(10 * power) && (LAZYLEN(grubs) < power * 2))
+				var/mob/living/simple_animal/hostile/redgrub/grub = new(src)// add new grubs if there's enough toxin for them
+				grub.food = 10
+				grubs += grub 
+				grub.togglehibernation()
+				grub.grubdisease = list(A)
+			if(prob(LAZYLEN(grubs) * (6/power)))// so you know its working. power lowers this so it doesnt spam you at high grub counts
+				to_chat(M, "<span class='warning'>You feel something squirming inside of you!</span>")
+			M.add_movespeed_modifier(MOVESPEED_ID_GRUB_VIRUS_SLOWDOWN, override = TRUE, multiplicative_slowdown = max(slowdown - 0.5, 0)) 
+
+/datum/symptom/parasite/End(datum/disease/advance/A)
+	. = ..()
+	var/mob/living/carbon/M = A.affected_mob
+	M.remove_movespeed_modifier(MOVESPEED_ID_GRUB_VIRUS_SLOWDOWN, TRUE)
+	for(var/mob/living/simple_animal/hostile/redgrub/grub in grubs)
+		M.visible_message("<span class='warning'>[M] vomits up a disgusting grub!</span>", \
+				"<span class='userdanger'>You vomit a large, slithering grub!</span>")
+		M.Stun((grub.growthstage * 10))
+		grub.forceMove(M.loc)
+		grub.togglehibernation()
+		playsound(M.loc, 'sound/effects/splat.ogg', 50, 1)
+
+/datum/symptom/parasite/OnDeath(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/carbon/M = A.affected_mob
+	for(var/mob/living/simple_animal/hostile/redgrub/grub in grubs)
+		grub.forceMove(M.loc)
+		grub.togglehibernation()
+		playsound(M.loc, 'sound/effects/splat.ogg', 50, 1)
+	if(isslimetarget(M))
+		for(var/I in 1 to (rand(1, A.stage)))
+			var/mob/living/simple_animal/hostile/redgrub/grub = new(M.loc)
+			grub.grubdisease = list(A)
+		M.gib()
+		M.visible_message("<span class='warning'>[M] is eaten alive by a swarm of red grubs!</span>")
+
+/datum/symptom/jitters
+	name = "Hyperactivity"
+	desc = "The virus causes restlessness, nervousness and hyperactivity, increasing the rate at which the host needs to eat,but making them harder to tire out"
+	stealth = -4
+	resistance = 0
+	stage_speed = 2
+	transmission = -3
+	level = 8
+	severity = 1
+	symptom_delay_min = 1
+	symptom_delay_max = 1
+	prefixes = list("Gray ", "Amped ", "Nervous ")
+	var/clearcc = FALSE
+	threshold_desc = "<b>Resistance 8:</b>The virus causes an even greater rate of nutriment loss, able to cause starvation, but its energy gain greatly increases<br>\
+					<b>Stage Speed 8:</b>The virus causes extreme nervousness and paranoia, resulting in occasional hallucinations, and extreme restlessness." 
+
+/datum/symptom/jitters/severityset(datum/disease/advance/A)
+	. = ..()
+	if(A.resistance >= 8)
+		severity -= 1
+	if(A.stage_rate >= 8)
+		severity -= 1
```

Comment:
because it heals stamina damage

Reply:
> The virus causes extreme nervousness and paranoia, resulting in occasional hallucinations, and extreme restlessness.

That sounds very negative to me, maybe it should be rephrased to sound positive then?

Reply:
I still don't like that this decreases severity while both causing negative symptoms and painting the thresholds as negative in the text.

Reply:
severity is a gamey number used to measure a virus' positive effects. giving this a positive severity would make it very easy to tack this onto high-severity diseases for disease stacking. I'm willing to raise it, but that will make the symptom itself a deal more powerful

Reply:
The thresholds could also not affect the severity, since the thresholds seem to have a balance of positive and negative effects. That said, my accent is on the description rather than the severity:

> extreme nervousness and paranoia, resulting in occasional hallucinations, and extreme restlessness

Nervousness, paranoia, hallucinations, extreme restlessness... Those all sound negative, and yet the effect is positive enough to decrease the severity?

Reply:
I updated the description to include the positive effects as well.